IDEAS/Buildings/Components/ZoneAirModels/BaseClasses/partial_AirLeakage.mo

```diff
@@ -0,0 +1,70 @@
+within IDEAS.Buildings.Components.ZoneAirModels.BaseClasses;
+partial model partial_AirLeakage "air leakage due to limied air tightness"
+
+  parameter Modelica.SIunits.Volume V "zone air volume";
+  parameter Real n50(min=0)=0.4 "n50-value of airtightness";
+  parameter Real n50toAch = 20 "Conversion factor from n50 to Air Change Rate";
```

Comment:
I'm wondering if this model should only contain m_flow_nominal as a parameter, instead of propagating `V`, `n50`, `n50toAch` etc. I think it makes more sense?